pipelines/applications-service-api/Dockerfile

```diff
@@ -15,8 +15,10 @@ ADD data data
 ADD package.json package.json
 ADD .sequelizerc .sequelizerc
 ADD package-lock.json package-lock.json
+ADD .eslintrc .eslintrc
 
 RUN npm i
```

Comment:
@cpcundill - not sure how is this happening, but this line is `RUN npm i` only ?